WordPress/gutenberg
# Site editor: Shorter screen reader announcement after changing pages

## What?


Partially fixes #56188

## Why?


The current title message is very verbose, this PR shortens it.

## How?


Adds a dedicated prop for accessibility announcements.

## Testing Instructions





1. Enable a screen reader.
2. Open the Appearance > Editor.
3. Notice the message spoken is shorter.
> Now displaying: template_name template_type

### Testing Instructions for Keyboard


## Screenshots or screencast
